### PR TITLE
feat: evm api for collection & item detail

### PR DESCRIPTION
**Pull request by preschian**
## Context

Provide api endpoint from onchain data for evm

## Ref

- [x] Related https://github.com/kodadot/nft-gallery/pull/10713

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Chore

### CODE REVIEW

File: ogi/server/utils/evm.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  type ChainOptions,
+} from 'thirdweb/chains'
+
+export const chains: {
+  [key: string]: Readonly<ChainOptions & { rpc: string }>
+} = {
+  base,
+  optimism,
+  avalanche,
+  polygon,
```
Comment (hassnian) [inline ogi/server/utils/evm.ts:16]: imx?

**Comment by vikiival** [comment]
Also support for mantle is on the way

File: ogi/server/utils/evm.ts (hunk 1 above)
Comment (preschian) [reply ogi/server/utils/evm.ts:62]: lets follow Prefix naming instead in here

**Comment by preschian** [comment]
added imx and mantle:
- imx: https://ogi-feat--evm-api-for-item-d.workers-ogi.pages.dev/api/evm/imx/collection/0xb2155a27832ae6d61e6041e13e8c7421ae025d64
- mantle: https://ogi-feat--evm-api-for-item-d.workers-ogi.pages.dev/api/evm/mantle/collection/0xd730afd953092e8032ee38fe8a3fa30b64c78014

**Review by Jarsen136 — APPROVED** [review]

**Review by hassnian — APPROVED** [review]

**Comment by preschian** [comment]
merging, and I will update the domain name here: https://github.com/kodadot/nft-gallery/pull/10713

**Pull request merged**